src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -152,3 +156,28 @@ const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | Man
         return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
   }
 };
+
+// Get validation errors that depend on an asynchronous request
+const getValidationErrors = async (appData: ApplicationData, req: Request): Promise<ValidationError[]> => {
+  const beneficialOwnersTypeEmptyNOCListErrors = await beneficialOwnersTypeEmptyNOCList(req, appData);
+
+  return [...beneficialOwnersTypeEmptyNOCListErrors];
+};
+
+const getPageRender = async (req: Request, res: Response, errors?: FormattedValidationErrors) => {
```

Comment:
Again, just `renderPage`? Although... this function either redirects or renders depending on some checks, so maybe `redirectOrRenderPage`?